ClinGen/clincoded
# Connects to #1082. Landing page demo access note.

**Steps to test:**
1) Add the following entries in the `/etc/hosts` file:
```
127.0.0.1    curation.clinicalgenome.org
127.0.0.1    curation-test.clinicalgenome.org
127.0.0.1    production.clinicalgenome.org
```
2) Go to **curation-test.clinicalgenome.org:6543** and expect to see the dark blue message in the light blue panel. Notice that the **"this"** in the "...explore _this_ text/demo version..." portion of the message.
3) Go to **curation.clinicalgenome.org:6543** and expect to see the dark blue message in the light blue panel. Notice that the **"a"** in the "...explore _a_ text/demo version..." portion of the message.
